packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -196,7 +196,17 @@ Lists all Custom Authorization Servers in this Okta organization
 
 ##### Request parameters
 
-None
+
+Parameter | Description                                                                                | Param Type | DataType | Required | Default
+--------- | ------------------------------------------------------------------------------------------ | ---------- | -------- | -------- | -------
+q         | Searches the `name` and `audiences` of Authorization Servers for matching value            | Query      | String   | FALSE    |
```

Comment:
making values plural

Searches the `name` and `audiences` of Authorization Servers for matching values